num2words/lang_AR.py

```diff
@@ -105,14 +102,16 @@ def to_currency(self, val, longval=True):
 to_ordnum = n2w.to_ordinal_num
 to_year = n2w.to_year
 
+
 def main():
-    for val in [ 1, 11, 12, 21, 31, 33, 71, 80, 81, 91, 99, 100, 101, 102, 155,
-             180, 300, 308, 832, 1000, 1001, 1061, 1100, 1500, 1701, 3000,
-             8280, 8291, 150000, 500000, 1000000, 2000000, 2000001,
-             -21212121211221211111, -2.121212, -1.0000100]:
+    for val in [1, 11, 12, 21, 31, 33, 71, 80, 81, 91, 99, 100, 101, 102, 155,
+                180, 300, 308, 832, 1000, 1001, 1061, 1100, 1500, 1701, 3000,
+                8280, 8291, 150000, 500000, 1000000, 2000000, 2000001,
+                -21212121211221211111, -2.121212, -1.0000100]:
         n2w.test(val)
-    n2w.test(1325325436067876801768700107601001012212132143210473207540327057320957032975032975093275093275093270957329057320975093272950730)
-    for val in [1,120,1000,1120,1800, 1976,2000,2010,2099,2171]:
+    n2w.test(
+        1325325436067876801768700107601001012212132143210473207540327057320957032975032975093275093275093270957329057320975093272950730)
+    for val in [1, 120, 1000, 1120, 1800, 1976, 2000, 2010, 2099, 2171]:
```

Comment:
I'm not sure about this change for the test with the really long number because the line is longer than the allowed line length for PEP8:

```
+ n2w.test(
+   1325325436067876801768700107601001012212132143210473207540327057320957032975032975093275093275093270957329057320975093272950730)
```

Reply:
Ohh the real fix is in the line 114 there were missing spaces before the "," I will fix the line 113.